lib/members_page.rb

```diff
@@ -0,0 +1,15 @@
+#!/bin/env ruby
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'scraped'
+
+class MembersPage < Scraped::HTML
+  decorator Scraped::Response::Decorator::AbsoluteUrls
+
+  field :member_rows do
+    noko.xpath('//table[@id="list"]//tr[td]').map do |tr|
+      (fragment tr => MemberRow).to_h
```

Comment:
This isn't necessarily _wrong_, but we tend to just return the `fragment` from these methods, and let the consumer decide if/when to use `to_h` — often `scraper.rb` might want to call methods on these fragments other than `to_h`.